.github/workflows/mirror.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Mirror master to main branches in the plugins repository.
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  mirror_job:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'flutter/plugins' }}
+    name: Mirror main branch to master branch
```

Comment:
Oops, I just noticed that this is backward. That's probably true of the flutter/packages version as well.

Reply:
updated the name.